<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signupPWD</class>
 <widget class="QDialog" name="signupPWD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>720</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="foreground">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>600</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_signup">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_acount">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_PWD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ENTER</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>foreground</zorder>
  <zorder>label_acount</zorder>
  <zorder>label_signup</zorder>
  <zorder>lineEdit_PWD</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
